tensorflow/core/common_runtime/eager/BUILD

```diff
@@ -263,6 +264,23 @@ cc_library(
     }),
 )
 
+cc_library(
+    name = "eager_op_rewrite_registry",
+    srcs = ["eager_op_rewrite_registry.cc"],
+    hdrs = ["eager_op_rewrite_registry.h"],
+    deps = [":eager_operation"],
+)
+
+tf_cc_test(
+    name = "eager_op_rewrite_registry_test",
+    srcs = ["eager_op_rewrite_registry_test.cc"],
+    deps = [
+        ":eager_op_rewrite_registry",
```

Comment:
This also needs to depend on something which exports the status header, like "//tensorflow/core:lib",

Reply:
Done. Thanks for letting me know.